docs/content/formulations/compressibleFlow/LES-Model.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: LES Model
+parent: Compressible Flow
+grand_parent: Formulations
+nav_order: 5
+---
+## Large-eddy Simulation of Compressible Flows
+
+### The filtered compressible Navier-Stokes equations can be expressed as (Hirch [1]):
+
+$$\begin{equation}\frac{\partial \bar{\rho}}{\partial t} + \frac{\partial(\bar{\rho} \tilde{u}_{j})}{\partial x_{j}} = 0\end{equation}$$
+
+$$\begin{equation}\frac{\partial (\bar{\rho} \tilde{u}_{i})}{\partial t} + \frac{\partial(\bar{\rho} \tilde{u}_{i}\tilde{u}_{j})}{\partial x_{j}} = -\frac{\partial \bar{p}}{\partial x_{i}} + \frac{\partial \mu \tilde{S}_{ij}}{\partial x_{j}} -
+ \frac{\partial \tau_{ij}}{\partial x_{j}}\end{equation}$$
+
+$$\begin{equation}\frac{\partial (\bar{\rho} \tilde{e})}{\partial t} + (\bar{\rho} \tilde{e}+\bar{p})\frac{\partial \tilde{u}_{j}}{\partial x_{j}} = \frac{\partial(\mu \tilde{S}_{ij}-\tau_{ij})\tilde{u}_{i}}{\partial x_{j}} + \frac{\partial }{\partial x_{j}}\left(\lambda\frac{\partial \tilde{T}}{\partial x_{j}}\right) - \frac{\partial{ q_{j}}}{\partial x_{j}}+ \bar{\dot{\omega}}_T\end{equation}$$
```

Comment:
I pulled the changes and rended the page.  It looks like we are missing some double $$ around some of the items.  Would you mind making sure they are all double $$.  Thanks!

<img width="934" alt="image" src="https://user-images.githubusercontent.com/75279702/184263279-e884dbe5-a02e-40a9-9a40-fcbcb5b0d123.png">